02-Logging/02-Saving-Array-Quantities.ipynb

```diff
@@ -45,11 +45,25 @@
   },
```

Comment:
<p>The rendered docs need to have no errors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>